user_reservation.py

```diff
@@ -0,0 +1,299 @@
+                user_end_date = datetime.strptime(input_end_date, "%Y-%m-%d").date()
+                if user_end_date >= user_beginning_date:
+                    user_end_date_str = user_end_date.strftime("%Y-%m-%d")
+                    counter = user_end_date - user_beginning_date
+                    days_counter = counter.days + 1
+                    print(f"Wybrano biurko w przedziale czasowym od {user_beginning_date} do {user_end_date}.")
+                    print(f"Całkowita liczba dni wynosi: {days_counter}")
+                    return user_end_date_str, user_end_date, days_counter
+                else:
+                    print("Data powinna być późniejsza niż data początkowa. Spróbuj ponownie.")
+            except ValueError:
+                print("Wprowadź odpowiednią datę w formacie YYYY-MM-DD")
+    user_end_date_str, user_end_date, days_counter = end_date(user_beginning_date)
+
+
+    def size_choosing():
+        print("Jakie biurko Cię interesuje?: ")
+        print("Biurko 1-osobowe - wprowadź cyfrę 1")
+        print("Biurko 4-osobowe - wprowadź cyfrę 4")
+        print("Biurko 10-osobowe - wprowadź cyfrę 10")
+        while True:
+            user_choice_size = input("Wprowadź swój wybór: ")
+            if user_choice_size.isdigit():
+                if int(user_choice_size) == 1:
+                    print(f"Twój wybór to biurko {user_choice_size}-osobowe.")
+                    return user_choice_size
+                elif int(user_choice_size) == 4:
+                    print(f"Twój wybór to biurko {user_choice_size}-osobowe.")
+                    return user_choice_size
+                elif int(user_choice_size) == 10:
+                    print(f"Twój wybór to biurko {user_choice_size}-osobowe.")
+                    return user_choice_size
+                else:
+                    print("Wprowadzono niewłaściwą wartość, podaj wartość cyfrą: 1, 4 lub 10.")
+            else:
+                print("Wprowadzono niewłaściwą wartość, podaj wartość cyfrą: 1, 4 lub 10.")
+    user_choice_size = size_choosing()
+
+
+    def display_choosing():
+        print("Czy interesuje Cię biurko z monitorem czy bez?")
+        print("Biurko z monitorem - wprowadź cyfrę 1")
+        print("Biurko bez monitora - wprowadź cyfrę 2")
+        while True:
+            user_choice_display = input("Wprowadź swój wybór: ")
+            if user_choice_display.isdigit():
+                if int(user_choice_display) == 1:
+                    return user_choice_display
+                elif int(user_choice_display) == 2:
+                    return user_choice_display
+                else:
+                    print("Wprowadzono niewłaściwą wartość, podaj 1 lub 2.")
+            else:
+                print("Wprowadzono niewłaściwą wartość, podaj wartość 1 lub 2.")
+    user_choice_display = display_choosing()
+
+    def presenting_avaliability():
+        if int(user_choice_size) == 1 and int(user_choice_display) == 1:
```

Comment:
całą tą funkcje mozna zrobic w 2 liniach 
```suggestion
       rozmiar = int(user_choice_size)
       monitor = int(user_choice_display) == 1
```
a nawet w jednej 
```suggestion
        return int(user_choice_size), int(user_choice_display) == 1
```